redwood-layout-testing/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'app.cash.redwood.generator.testing'
```

Comment:
I'm on the fence about making every schema generate this.

Currently, we only make you generate the composables and the widget interfaces (and layout modifiers, but only if needed).

For specialized implementations of the widgets such as the protocol, we currently generate all of the needed widget subtypes within that schema's package. We could do that here.

One difference is that the protocol only has a single public type whereas this testing API has one value type for each widget. That would mean things like `ColumnValue` would show up as `com.example.myschema.testing.redwoodlayout.ColumnValue` instead of `app.cash.redwood.layout.testing.ColumnValue`. I suspect that might be okay.

I would feel a lot more comfortable if I got to play around with automatically propagating these dependencies. We can leave this as-is for now and I"ll play around that sometime later (and file an issue).

Reply:
I could see us wanting to share test utilities for layouts & lazy layouts. I’m not yet sure what that’ll be, but it’s only practical if they’re shared.